docs/modules/developer-guides/pages/deprecating-steps.adoc

```diff
@@ -17,16 +17,16 @@ For deprecation of Java code steps use annotation `@Replacement` with the follow
 ./src/main/java/com/mycompany/steps/MySteps.java
 [source,java]
 ----
-    @Deprecated(since = "0.5.0", forRemoval = true)
-    @Replacement(versionToRemoveStep = "0.6.0", replacementFormatPattern = "Given the step with parameters `%2$s` and `%1$s`")
-    @Given(value = "the step with parameters '$parameter1' and '$parameter2'", priority = 1)
-    public void givenStep(String parameter1, String parameter2)
-    {
-        // ... step logic
-    }
+@Deprecated(since = "0.6.0", forRemoval = true)
+@Replacement(versionToRemoveStep = "0.7.0", replacementFormatPattern = "Given the step with parameters `%2$s` and `%1$s`")
```

Comment:
can we refer some variables here so we do not need to update this place every release?

Reply:
do you know such variables?

Reply:
it doesn't

Reply:
ChatGPT?